test/models/character_test.exs

```diff
@@ -0,0 +1,149 @@
+defmodule Yggdrasil.CharacterTest do
+  use Yggdrasil.ModelCase
+
+  alias Yggdrasil.User
+  alias Yggdrasil.Game
+  alias Yggdrasil.Character
+
+  @min_len 3
+
+  @user1   %{username: "tester",
+             password: "test123", 
+             password_confirmation: "test123" }
+
+  @user2   %{username: "tester2",
+             password: "test123", 
+             password_confirmation: "test123" }
+
+  @game1   %{name: "abcdef", description: "abcdefghi"}
+  @game2   %{name: "ghijkl", description: "abcdefghi"}
+
+  @character %{name: "foobar", user_id: nil, game_id: nil}
```

Comment:
The name is a valid length and serves as the basis for all the tests that center around valid changesets and the combos where the game_id/user_id fire constraint errors. In those cases the name needs to be correct but the until you hit the DB you don't know if the foreign keys are correct for example.

Given that I refrained from calling it a valid_character, but perhaps there is better name - open for suggestions.